html/includes/output/query.inc.php

```diff
@@ -43,7 +43,7 @@
                 $rule['query'] = GenSQL($rule['rule']);
             }
             $sql = $rule['query'];
-            $qry = dbFetchRow($sql, array($device_id));
+            $qry[] = dbFetchRow($sql, array($device_id));
```

Comment:
Won't this work better `$qry = dbFetchRows($sql, array($device_id));`?
`Row`->`Rows`